docs/QUICK_START.md

```diff
@@ -0,0 +1,201 @@
+
+If you are using one of the MatekSys Tx module kits, print or purchase a JR bay case from the STL download available from [MatekSys](https://www.mateksys.com/?portfolio=mr900-30-tx#tab-id-1).
+
+If you will use the MatekSys HC04 module to connect your ground station via Bluetooth, set all three [dip switches](https://MATEKSYS.md#tx-module-hc-04-bluetooth-notes) to the on position before assembling your Tx module.
+
+If you are using the [R9M Tx module](FRSKY_R9.md#r9m-tx-module):
+- Set dip switch 1 to the On position to configure it for serial data
+- Install the [Arduino IDE](https://www.arduino.cc/en/software) and use it to build and flash [firmware](WIRELESS_BRIDGE.md) on your ESP32 module.  Be sure to select the R9M variant for your module when you edit the mlrs-wireless-bridge.ino.
+- Connect your [ESP32 wireless bridge](https://FRSKY_R9.md#esp32-wireless-bridge) module to the serial and power pins of your R9M.
+
+When choosing and connecting antennas be aware of the difference between SMA and RP-SMA.  It is, unfortunately, possible to attach an RP-SMA antenna to an SMA jack but there will be no electrical connection of the signal.
+
+## 3. Connect your receiver to your flight controller
+
+For the 1 watt MatekSys [mLRS receivers](https://www.mateksys.com/?portfolio=mr24-30#tab-id-2):
+- connect the receiver rx1 to a FC UART Tx.
+- connect the receiver tx1 to the corresponding FC UART rx.
+- Connect the receiver V pin to a 5 volt output on your FC.
+- Connect the receiver G pin to a ground pad on your FC.
+- If you will use CRSF for RC channels, connect the receiver TX2 to a different UART rx on your FC.
+
+For most ELRS receivers, the [connections](ELRS_RECEIVERS.md#connections) are similar to those above except CRSF is not available.
+
+Directly connect your ground station to your ArduPilot flight controller via a USB cable and configure the serial port (x) you wired to your receiver serial port.  Don't forget to press "write parameters" if your ground station requires this step.
+- Set SERIALx\_PROTOCOL to "MAVLink2" (2).
+- Set SERIALx\_BAUD to "57600" (57) (default mLRS Rx Ser Baudrate).
+
+If you will use CRSF or SBUS for RC channels, use your ground station to configure the serial port (x) you wired to your receiver RC output.
+- Set SERIALx_PROTOCOL to "RCIN" (23).
+- Set RC_PROTOCOLS to 536.
+- Set RSSI_TYPE to 3.
+
+## 4. Install firmware if required
+
+If both your receiver and Tx modules came pre-installed with mLRS firmware as with the MatekSys mLRS modules, you can use the existing firmware unless you need a bug fix or feature which was added in a newer firmware version.
+
+Carefully install your Tx module into the bay of your EdgeTX or OpenTX radio.  Some Tx modules can be fussy about pin alignment;  don't use excessive force or you may break something.
+
+[Flashing firmware](https://www.mateksys.com/?portfolio=mr24-30-tx#tab-id-3) to the MatekSys mLRS Tx modules is especially easy.
```

Comment:
Shouldn't this be Tx Module / Receiver agnostic?  For example:

[Flashing firmware](https://www.mateksys.com/?portfolio=mr24-30-tx#tab-id-3) to the MatekSys mLRS hardware is especially easy.

Also, I think it would make sense to link to the mLRS Web Flasher (to avoid having to install CubeProgrammer): https://mlrs.xyz/flash/